cabot/cabotapp/tests/tests_basic.py

```diff
@@ -859,3 +859,29 @@ def test_get_all_duty_officers(self):
                             (officers[1][0].username, officers[1][1][0].name)]
         self.assertIn(('dolores@affirm.com', 'Principal'), officer_schedule)
         self.assertIn(('maeve@affirm.com', 'Secondary'), officer_schedule)
+
+
+class TestUserPermissions(LocalTestCase):
+    def test_user_profile_permissions(self):
+        user1 = User.objects.create_user(username='user1', password='pass1')
```

Comment:
thanks for doing the tests! Can we split them into separate test cases for superuser and non-superuser though (set up the users in setUp(), then we can use them in both tests).

In general I think it's good to have each test testing one specific thing (e.g. user can't access superuser's profile) because then you can tell exactly what failed from just the test name instead of having to dig through the code to see which line failed + you can tell exactly what failed (if the first assert failed in this test, we don't know whether the other ones would have passed or not)